This patch adds MATE and Openbox "OnlyShowin" desktop file support.
diff -Naur xdg-autostart-0.1/libxdgutils/XDGMenu.cpp xdg-autostart-0.1.new/libxdgutils/XDGMenu.cpp
--- xdg-autostart-0.1/libxdgutils/XDGMenu.cpp	2014-01-01 02:11:32.445630176 +0200
+++ xdg-autostart-0.1.new/libxdgutils/XDGMenu.cpp	2014-01-01 02:12:32.202634018 +0200
@@ -189,6 +189,8 @@
     "LXDE",
     "ROX",
     "XFCE",
+    "MATE",
+    "OPENBOX",
     "Old",
     NULL
 };
